core-services/image-mirroring/openshift/mapping_origin_4_3

```diff
@@ -192,6 +192,7 @@ registry.svc.ci.openshift.org/origin/4.3:ptp quay.io/openshift/origin-ptp:4.3 qu
 registry.svc.ci.openshift.org/origin/4.3:ptp-operator quay.io/openshift/origin-ptp-operator:4.3 quay.io/openshift/origin-ptp-operator:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:rbac-permissions-operator quay.io/openshift/origin-rbac-permissions-operator:4.3 quay.io/openshift/origin-rbac-permissions-operator:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:recycler quay.io/openshift/origin-recycler:4.3 quay.io/openshift/origin-recycler:4.3.0
+registry.svc.ci.openshift.org/origin/4.3:route-override-cni quay.io/openshift/origin-route-override-cni:4.3 quay.io/openshift/origin-route-override-cni:4.3.0
```

Comment:
What does this do?

Reply:
Provides a CNI plugin -- route-override-cni provides a way to override routing within pods, especially for use with additional network attachments, and allows users to express that intent to do so in NetworkAttachmentDefinitions.

Reply:
The use of the word `route` is problematic because we have a top level Openshift concept about routes.  It would be more correct for this to be scoped to the subcomponent area in a way that removes the potential for confusion (since we have already given top level `router` to the openshift component).  `network-route-override-cni` might be ok, or `multus-route-override-cni` might also be acceptable.